src/nodedef.cpp

```diff
@@ -890,13 +890,13 @@ void CNodeDefManager::updateTextures(IGameDef *gamedef,
 				for (u32 i = 0; i < 6; i++)
 					tiledef[i].name += std::string("^[noalpha");
 			}
-			if (f->waving == 1)
+			if (f->waving >= 1)
 				material_type = TILE_MATERIAL_WAVING_LEAVES;
 			break;
 		case NDT_PLANTLIKE:
 			f->solidness = 0;
 			f->backface_culling = false;
-			if (f->waving == 1)
```

Comment:
Perhaps the docs should be adjusted then, they aren't really clear in this point. If you want a plantlike node to wave like leaves, you'd set waving = 2. Best would be if the docs are formulated in a way that doesn't allow this misunderstanding.